nuxeo-features/nuxeo-admin-center/nuxeo-admin-center-core/src/main/java/org/nuxeo/ecm/admin/NuxeoCtlManager.java

```diff
@@ -62,44 +57,29 @@ private static String winEscape(String command) {
     }
 
     protected static boolean doExec(String path, String logPath) {
-        String[] cmd;
-        if (SystemUtils.IS_OS_WINDOWS) {
-            cmd = new String[] { "cmd", "/C", winEscape(new File(path, CMD_WIN).getPath()), "--gui=false", "restartbg" };
-        } else {
-            cmd = new String[] { "/bin/sh", "-c", "\"" + new File(path, CMD_POSIX).getPath() + "\"" + " restartbg" };
-        }
-
-        Process p1;
         try {
+            String[] cmd = getCommand(path);
             if (log.isDebugEnabled()) {
                 log.debug("Restart command: " + StringUtils.join(cmd, " "));
             }
-            ProcessBuilder pb = new ProcessBuilder(cmd);
-            p1 = pb.start();
+            ProcessBuilder pb = new ProcessBuilder(cmd).
+                    redirectOutput(new File(logPath, "restart.log")).
+                    redirectError(new File(logPath, "restart-err.log"));
```

Comment:
The three lines below are not correctly formatted. `.` should be at the beginning of new lines.